tests/integration/test_writes.py

```diff
@@ -355,6 +355,26 @@ def test_data_files(spark: SparkSession, session_catalog: Catalog, arrow_table_w
     assert [row.deleted_data_files_count for row in rows] == [0, 0, 1, 0, 0]
 
 
+@pytest.mark.integration
+def test_multiple_spark_sql_queries(spark: SparkSession, session_catalog: Catalog, arrow_table_with_null: pa.Table) -> None:
```

Comment:
Nit: I think a better name would be `test_python_writes_with_spark_snapshot_reads` or something more specific than what it currently is . It's mre verbose but I think it captures the goal of the test better

Reply:
make sense, added!